src/main/java/com/notification/repo/PasswordResetTokenDetailsRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.notification.repo;
+
+import com.notification.document.PasswordResetTokenDetails;
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+public interface PasswordResetTokenDetailsRepository extends MongoRepository<PasswordResetTokenDetails, ObjectId> {
+    boolean existsByUserEmail(String email);
```

Comment:
I don't think this is required here, as user can forgot password as many times he wants.

Reply:
Yes removed it.